tests/avocado/test_ovmf_with_240_vcpus.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import sys
+
+from avocado import Test
+from base import VM
+
+
+class TestOvmfVcpus(Test):
+    """
+    :avocado: tags=ovmf
+    """
+
+    def setUp(self):
```

Comment:
If we still don't have dependency-tags or something like `checkDeps()` (that would run before `setUp()`), then we could use decorators, IIRC.

My point is that a test that runs `shutil.copy('/usr/share/edk2/ovmf/OVMF_VARS.fd' ...` will FAIL with errors most of the time in the context of upstream.